src/ResourceGraph/ResourceGraph/help/Search-AzGraph.md

```diff
@@ -64,34 +66,43 @@ A complex query on resources featuring field selection, filtering and summarizin
 
 ### Example 3
 ```powershell
-PS C:\> Search-AzGraph -Query 'project id, name' -SkipToken 'skiptokenvaluefromthepreviousquery=='
+PS C:\> $response = Search-AzGraph -Query "project id, name, type, location" -First 2
+PS C:\> Search-AzGraph -Query "project id, name, type, location" -SkipToken $response.SkipToken
 
 
-id         : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/Service-INT-b/providers/Microsoft.Compute/virtualMachineScaleSets/nt2
-name       : nt2
+id         : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/test/providers/Microsoft.Network/networkInterfaces/17ni
+name       : 17ni
+type       : microsoft.network/networkinterfaces
+location   : westeurope
+ResourceId : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/test/providers/Microsoft.Network/networkInterfaces/17ni
 
-id         : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/Service-INT-b/providers/Microsoft.EventGrid/topics/egtopic-2
-name       : egtopic-2
+id         : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/test/providers/Microsoft.Network/networkSecurityGroups/17nsg
+name       : 17nsg
+type       : microsoft.network/networksecuritygroups
+location   : westeurope
+ResourceId : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/test/providers/Microsoft.Network/networkSecurityGroups/17nsg
 ```
 
-A query with the skip token passed from the previous query results
+A query with the skip token passed from the previous query results. Please note that keeping id in the results is mandatory to get back a skip token.
 
 ### Example 4
 ```powershell
-PS C:\> Search-AzGraph -Query 'project id, name, type, location, tags' -First 2 -ManagementGroup 'MyManagementGroupId' -AllowPartialScope
+PS C:\> Search-AzGraph -Query "project id, name, type, location, tags" -First 2 -ManagementGroup MyManagementGroupId -AllowPartialScope
 
 
 id         : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/Service-INT-a/providers/Microsoft.Compute/virtualMachineScaleSets/nt
 name       : nt
 type       : microsoft.compute/virtualmachinescalesets
 location   : eastus
 tags       : @{resourceType=Service Fabric; clusterName=gov-art-int-nt-a}
+ResourceId : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/Service-INT-a/providers/Microsoft.Compute/virtualMachineScaleSets/nt
 
 id         : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/Service-INT-a/providers/Microsoft.EventGrid/topics/egtopic-1
 name       : egtopic-1
 type       : microsoft.eventgrid/topics
 location   : westus2
 tags       :
+ResourceId : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/Service-INT-a/providers/Microsoft.EventGrid/topics/egtopic-1
 ```
 
 A query scoped to the management group that allows the query to succeed with partial scope result if MyManagementGroupId has more than N subscriptions underneath.
```

Comment:
>more than N [](http://example.com/codeflow?start=125&length=11)

this statement feels keeping it open question to end customers thinking about what is the N means for my tenant. N => 1000 or 5000.

Should we add information on - 

if the given management group exceeds the limit of number of allowed subscriptions while querying for resources. Also provide information that while querying for container resources we don't have any limits.